app/components-react/windows/Notifications.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react';
+import moment from 'moment';
+import cx from 'classnames';
+import { useRenderInterval, useVuex } from 'components-react/hooks';
+import { ModalLayout } from 'components-react/shared/ModalLayout';
+import { Services } from 'components-react/service-provider';
+import { INotification } from 'services/notifications';
+import { $t } from 'services/i18n';
+import styles from './Notifications.m.less';
+
+export default function Notifications() {
+  const { NotificationsService } = Services;
+
+  useRenderInterval(() => {}, 60 * 1000);
```

Comment:
Possible to add a comment explaining what this render interval is for?  Had to look up in old code and I guess it's to force a re-render of the relative timestamps.